frontend-nannyway/src/components/header/NavMenuSecondaryColor.js

```diff
@@ -0,0 +1,55 @@
+import {Button, Menu, MenuItem} from "@material-ui/core";
+import React from "react";
+import {Link} from "react-router-dom";
+import {makeStyles} from "@material-ui/core/styles";
+import myTheme from "../../styling/muiTheme";
+
+const useStyles = makeStyles((theme) => ({
+    darkColor: {
+        color: myTheme.palette.secondary.dark,
+    },
+    buttonMenu: {
+        display: "flex",
+        justifyContent: "center",
+    }
+}));
+
+export default function NavMenuSecondaryColor() {
+    const classes = useStyles();
+
+    const [anchorEl, setAnchorEl] = React.useState(null);
+
+    const handleClick = (event) => {
+        setAnchorEl(event.currentTarget);
+    };
+
+    const handleClose = () => {
+        setAnchorEl(null);
+    };
+
+    return (
+        <div className={classes.buttonMenu}>
+            <Button color={"secondary"} variant={"contained"} aria-controls="simple-menu" aria-haspopup="true"
```

Comment:
```suggestion
            <Button color={{colorScheme}} variant={"contained"} aria-controls="simple-menu" aria-haspopup="true"
```

Now in combination with the suggestion in `line 17` you can just pass a string "primary" or "secondary" when you want to render `<NavMenuSecondaryColor>`, like you do in `frontend-nannyway/src/components/header/AppBarPrimaryColor.js` in `line 36`, which would look like this: `<NavMenuSecondaryColor colorScheme="primary" >`
That way you can use the same component (name suggestion: `<NavMenu>`) with two different colors and have one less file 🎉

Reply:
Done... with a little help of my friend 🌷 😃